hibernate-core/src/main/java/org/hibernate/dialect/MariaDBDialect.java

```diff
@@ -166,6 +162,12 @@ boolean supportsAliasLocks() {
 		return false;
 	}
 
+	// https://jira.mariadb.org/browse/MDEV-19078
+	@Override
+	public boolean supportsLateral() {
+		return false;
+	}
+
```

Comment:
This seems to be necessary now to fix some tests in HQLTests that failed for MariaDB.